plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintPlugin.kt

```diff
@@ -406,4 +386,24 @@ open class KtlintPlugin : Plugin<Project> {
     private inline fun <reified T> Convention.getPluginHelper() = getPlugin(T::class.java)
 
     private fun Project.isConsolePlain() = gradle.startParameter.consoleOutput == ConsoleOutput.Plain
+
+    private inner class PluginHolder(
```

Comment:
I don't think this needs to be an `inner` class. Using the `inner` is basically the opposite of using `static` on a nested class in Java.

Unless you are scoping this class over something I'm not seeing, I think you can remove it.

Reply:
Removed `inner` and force-pushed.